src/main/java/io/openliberty/tools/common/plugins/util/BinaryScannerUtil.java

```diff
@@ -102,62 +105,65 @@ public BinaryScannerUtil(File binaryScanner) {
      *                                      features that should work to run the application
      */
     public Set<String> runBinaryScanner(Set<String> currentFeatureSet, List<String> classFiles, Set<String> allClassesDirectories,
-            String eeVersion, String mpVersion, boolean optimize)
+            String logLocation, String eeVersion, String mpVersion, boolean optimize)
             throws PluginExecutionException, NoRecommendationException, RecommendationSetException, FeatureModifiedException {
         Set<String> featureList = null;
         if (binaryScanner != null && binaryScanner.exists()) {
             try {
-                Method driveScanMavenFeatureList = getScannerMethod();
-                String[] binaryInputs = getBinaryInputs(classFiles, allClassesDirectories, optimize);
-                List<String> currentFeatures = new ArrayList<String>(currentFeatureSet);
+                Method generateFeatureSetMethod = getScannerMethod();
+                // names: binaryInputs, targetJavaEE, targetMicroProfile, currentFeatures, logLocation, logLevel, locale
+                Set<String> binaryInputs = getBinaryInputs(classFiles, allClassesDirectories, optimize);
+                String logLevel;
+                if (isDebugEnabled()) {
+                    logLevel = "*=FINE";  // generate messages for debugging by support team
+                } else {
+                    logLevel = null;
+                    logLocation = null;
+                }
                 debug("Calling " + binaryScanner.getName() + " with the following inputs...\n" +
-                      "  binaryInputs: " + Arrays.toString(binaryInputs) + "\n" +
-                      "  eeVersion: " + eeVersion + "\n" +
-                      "  mpVersion: " + mpVersion + "\n" +
-                      "  currentFeatures: " + currentFeatures + "\n" +
-                      "  locale: " + java.util.Locale.getDefault());
+                        "  binaryInputs: " + binaryInputs + "\n" +
+                        "  eeVersion: " + eeVersion + "\n" +
+                        "  mpVersion: " + mpVersion + "\n" +
+                        "  currentFeatures: " + currentFeatureSet + "\n" +
+                        "  logLocation: " + logLocation + "\n" +
+                        "  logLevel: " + logLevel + "\n" +
+                        "  locale: " + java.util.Locale.getDefault());
                 debug("The following messages are from the application binary scanner used to generate Liberty features");
```

Comment:
What sort of messages can be returned from the binary scanner that we do not catch in the exceptions below or in the logs?

Reply:
In prior versions the binary scanner could print messages. In the second most recent jar it would print messages if there were no logs directory but the latest jar has fixed this. Do you think we should remove these debug statements now?

Reply:
Thanks Paul, in that case I think these statements should be removed.

Reply:
Ok